public/app/store/configureStore.ts

```diff
@@ -22,11 +21,17 @@ export function configureStore() {
     },
   });
 
-  const middleware = process.env.NODE_ENV !== 'production' ? [toggleLogActionsMiddleware, thunk, logger] : [thunk];
+  const middleware = process.env.NODE_ENV !== 'production' ? [toggleLogActionsMiddleware, logger] : [];
+
+  const reduxDefaultMiddlewars = getDefaultMiddleware<StoreState>({
```

Comment:
A small typo, should be `reduxDefaultMiddlewares`or just `reduxDefaultMiddleware`.